flags.go

```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 )
 
+const envVarDelimiter = "\n"
```

Comment:
Good question. I think at the very least it should be a public variable, but ideally it would be configurable per-parser. That's a bit more involved though, as the application isn't accessible to flag clauses.